docker-compose.yml

```diff
@@ -1,11 +1,11 @@
 services:
-  gunicorn:
-    container_name: gunicorn
-    build: Services/Gunicorn
+  guni_game:
+    container_name: guni_game
+    build: Services/Gunicorn_Game
     restart: always
     env_file: .env
     volumes:
-      - django_data:/app/Project
+      - Game:/app/Project
     depends_on:
       postgres:
```

Comment:
need change to game_db instead of database

Reply:
> need change to game_db instead of database

I don't understand what i have to do here.

Reply:
The depends on is not on the right database container, for game is game_db or something like that.